src/site/includes/breadcrumbs.drupal.liquid

```diff
@@ -12,7 +12,15 @@
 
     {% for crumb in crumbs %}
       {% if forloop.last == true %}
-        <li><a onclick="recordEvent({ 'event': 'nav-breadcrumb', 'breadcrumb-click-label': '{{ crumb.text }}', 'breadcrumb-click-level': '{{ forloop.index }}', 'breadcrumb-total-levels': '{{ crumbs.length }}' });" aria-current="page" href="/{{ path }}">{{ crumb.text }}</a></li>
+        <li>
+            <a 
+              class="breadcrumb-link" 
+              onclick="recordEvent({ 'event': 'nav-breadcrumb', 'breadcrumb-click-label': '{{ crumb.text }}', 'breadcrumb-click-level': '{{ forloop.index }}', 'breadcrumb-total-levels': '{{ crumbs.length }}' });" 
+              aria-current="page" 
+              href="/{{ path }}">
+              {{ crumb.text }}
```

Comment:
Could we add the appropriate `lang` attribute here instead of having to do it in `configureBreadCrumbLinks`?

Reply:
i wish... since the drupal template is already rendered, and the language depends on a user action / toggle, it won't be possible without JS. even more of a reason to move towards react for i18n.

Reply:
we could read from `window.location` in the template actually, via a liquid filter.

Reply:
But the toggle just takes the user to a different page, doesn't it? So couldn't we do something like -

```liquid
{% assign lang = 'en' %}
{% if path == '/coronavirus-veteran-frequently-asked-questions-esp' %}
  {% assign lang = 'es' %}
{% endif %}

<!-- page.drupal.liquid -->
<body lang="{{ lang }}">
```

I think we'd want to do something more sophisticated then an exact check on the URL but seems like this is something doable in the templates

Reply:
yeah... i think these are definitely good points for a broader discussion around i18n... whether to do it via js, in templates, or via react. i will try that approach with just the breadcrumbs before merging

Reply:
for the breadcrumbs, we can also add the lang attribute in the cms directly